lib/vbridge/VBridge.ts

```diff
@@ -163,7 +163,7 @@ class VBridge {
   //   return new Bridge(bridgeConfig.bridgeSides, bridgeConfig.webbTokenAddresses, linkedAnchors, bridgeConfig.anchors);
   // }
 
-  public static async deployVBridge(vBridgeInput: VBridgeInput, deployers: DeployerConfig): Promise<VBridge> {
+  public static async deployVBridge(vBridgeInput: VBridgeInput, deployers: DeployerConfig, tokenInstance?: GovernedTokenWrapper): Promise<VBridge> {
```

Comment:
I added `tokenInstance` as an optional input to `deployBridge`, is this what you had in mind?